internal/pkg/ecs/ecs.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/copilot-cli/internal/pkg/aws/ecs"
+	awsECS "github.com/aws/copilot-cli/internal/pkg/aws/ecs"
```

Comment:
aren't these two packages the same? or am I not reading properly 👀

Reply:
🤦‍♂️i'll get rid of it